Misc/NEWS.d/next/Security/2023-01-24-16-12-00.gh-issue-101283.9tqu39.rst

```diff
@@ -0,0 +1,6 @@
+Windows version of :meth:`subprocess.Popen.__init__` with
+``shell=True`` made its shell search algorithm protected from dropping
+a program named ``cmd.exe`` into a current directory. Now the current
+directory is used as the last resort if system environment variables
+are totally stripped and Windows is installed not into ``C\Windows``.
+Patch by Eryk Sun.
```

Comment:
It seemed important to Oleg to credit me, so I left it in place in the suggested revision. It wasn't my goal in making suggestions to take credit. I don't expect them to be accepted verbatim, or even accepted at all. They can be resolved as rejected or with a completely new take on the problem. :shrug:

Reply:
@zooba Eryk greatly expanded my original fixed-path-only approach effectively replacing it with the dynamic one. I see it as a different implementation of the same idea so that's why I've attributed him.

On a second thought, though, I agree that totally omitting myself as an author of the original patch was not the right way, so I've added myself back.